integrated_channels/blackboard/models.py

```diff
@@ -146,6 +147,7 @@ def encrypted_client_id(self, value):
     client_secret = models.CharField(
         max_length=255,
         blank=True,
+        null=True,
         default='',
```

Comment:
Any reason we are defining default when `null=True` and `blank=True`.

Reply:
[REFERENCE](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Howtodropacolumn)

@zamanafzal because of the feedback i got here https://github.com/openedx/edx-platform/pull/34912

Reply:
default='' in this context seems redundant since you already have blank=True and null=True. I guess it can be removed now